pkg/scheduler/apis/config/v1beta2/conversion_test.go

```diff
@@ -35,11 +34,7 @@ func TestV1beta2ToConfigKubeSchedulerConfigurationConversion(t *testing.T) {
 		{
 			name:   "default conversion v1beta2 to config",
 			config: v1beta2.KubeSchedulerConfiguration{},
-			want: config.KubeSchedulerConfiguration{
-				AlgorithmSource: config.SchedulerAlgorithmSource{
-					Provider: pointer.StringPtr(v1beta2.SchedulerDefaultProviderName),
-				},
-			},
+			want:   config.KubeSchedulerConfiguration{},
```

Comment:
Just remove this file. The important bits are covered in scheme/scheme_test.go,

same for v1beta1

Reply:
done.